.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I added some of the linting config from our FED docs, but then I got scared when it changed the JS files a whole bunch, didn't want to have a noisy commit.

Reply:
yeah was thinkin bout this; @davist11, any opposition to these settings? we can have a separate commit formattin the repo

Reply:
appreciate you bringing these in tho 👍

Reply:
Umm, nope no opposition. There's def a complexity line where we figure out if we need to bring a whole build process in...but for right now it's probably overkill.

Do we need the jest stuff in here yet? Or does it cause no harm?